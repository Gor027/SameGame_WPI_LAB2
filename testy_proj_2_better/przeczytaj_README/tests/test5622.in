 J   ^ J^^JJ^
J^ ^ ^^^^ J^^ ^^^^ ^JJ
J JJJ J^^ JJ^ JJ
^ ^J ^   J^J ^ J^^ ^  ^JJJJ^   ^  J  ^J J JJJ
^ ^   J^J ^J^^JJJ J J^^   J^J^J^ J JJ^J ^ ^^^JJJJ ^J JJ ^^^^
^^J ^^^
^ ^^ ^ JJ^ J^JJJ JJ ^JJJ  ^^ J^^^J ^ JJJ ^  JJ JJ^^  J^ JJ JJ^^  ^JJ ^ J^
^   ^^^J^^ ^^JJ^J^^  ^^J  JJJ ^ 
J J  JJ^J ^JJJ^JJ    J^ J J^J 
^J ^^J ^JJ^^ J^    J ^ J ^^    ^^ ^^  JJ^JJ  J JJJ^^J  ^^   J  JJ^ ^ ^J J^J
 ^ JJ^J
 J^ J  ^JJJ JJ^^^    ^^ ^ ^J^ JJ^^JJ  JJJ ^ ^^ JJJJJJJ 
