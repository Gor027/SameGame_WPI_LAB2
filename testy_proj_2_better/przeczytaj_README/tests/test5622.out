     ^ J^  J
J    ^ ^^ J^^ ^^
J J  ^^^^ JJ^^JJ     J    J
^J^^J^J^J^JJ^^JJ^    ^^   J^
^^^J^^JJJJ^J^JJJ^    J^   ^^   ^      J
^^JJ^^^J^^^J^JJJJ^   ^JJ  J^J J^   J  J J J JJ JJ ^
^J^^^J^^J^^^^J^^J^J^JJ^JJ JJJ ^^^JJ^ ^JJ^ ^ ^J JJ^^J J  ^
^^J^JJJ^JJ^JJJ^JJJJ^J^^JJJ^^J^^^^J^^ ^JJ^^JJJJ JJJ^^JJJ J^ ^J    ^ J^  ^
JJ^JJ^J^JJJ^JJ^^^^^J^^^J^^^J^JJJ^^JJJJJJJJ^^^^JJJJJJJJ^^^J^JJ^^JJ^J^^JJJ^J
